daterangepicker.js

```diff
@@ -643,7 +643,12 @@
             if (dayOfWeek == this.locale.firstDay)
                 startDay = daysInLastMonth - 6;
 
+            const maxDefinedYear = 9999;
+            const minDefinedYear = 1899;
             var curDate = moment([lastYear, lastMonth, startDay, 12, minute, second]);
```

Comment:
I would add these two as options so that if they're not supplied, this behaviour is disabled. Also it will allow us to change the values easily where we use it.